package.json

```diff
@@ -62,28 +63,41 @@
   },
   "private": true,
   "preferGlobal": false,
+  "c8": {
+    "all": true,
+    "reporter": [
+      "lcov",
+      "text"
+    ],
+    "include": [
+      "add-on/src/**"
+    ]
+  },
   "devDependencies": {
     "@babel/core": "7.17.5",
     "@babel/plugin-proposal-class-properties": "7.16.7",
     "@babel/preset-env": "7.16.11",
-    "babel-loader": "8.2.3",
+    "@istanbuljs/esm-loader-hook": "^0.2.0",
+    "abort-controller": "^3.0.0",
+    "babel-loader": "^8.2.5",
     "babel-plugin-syntax-async-generators": "6.13.0",
+    "c8": "^7.12.0",
     "chai": "4.3.6",
     "cross-env": "7.0.3",
-    "css-loader": "6.6.0",
+    "css-loader": "^6.7.1",
     "download-cli": "1.1.1",
     "file-loader": "6.2.0",
     "fs-promise": "2.0.3",
     "get-firefox": "5.0.0",
-    "ignore-loader": "0.1.2",
+    "ignore-loader": "^0.1.2",
     "ignore-styles": "5.0.1",
     "json": "11.0.0",
     "mem-storage-area": "1.0.3",
     "mini-css-extract-plugin": "2.5.3",
-    "mocha": "9.2.0",
+    "mocha": "9.2.2",
     "npm-run-all": "4.1.5",
-    "nyc": "15.1.0",
-    "raw-loader": "4.0.2",
+    "path": "^0.12.7",
```

Comment:
NIT: I usually pin my dependencies (don't use `^`), npm packages are known to frequently push breaking changes and not correctly follow semver. https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/

I prefer to use a [dependency manager](https://github.com/dependabot)

Reply:
seems like a blocker needing addressed prior to merge.